demo/admin/src/products/ManufacturerFilter.tsx

```diff
@@ -0,0 +1,79 @@
+import { gql, useQuery } from "@apollo/client";
+import { InputBase } from "@mui/material";
+import Autocomplete from "@mui/material/Autocomplete";
+import { GridFilterInputValueProps, GridFilterOperator } from "@mui/x-data-grid-pro";
+import * as React from "react";
+import { useIntl } from "react-intl";
+
+import { GQLManufacturersFilterQuery, GQLManufacturersFilterQueryVariables } from "./ManufacturerFilter.generated";
+
+const manufacturersQuery = gql`
+    query ManufacturersFilter($offset: Int!, $limit: Int!, $search: String) {
+        manufacturers(offset: $offset, limit: $limit, search: $search) {
+            nodes {
+                id
+                name
+            }
+            totalCount
+        }
+    }
+`;
+
+// Source: https://mui.com/x/react-data-grid/filtering/customization/#multiple-values-operator
+function ManufacturerFilter({ item, applyValue }: GridFilterInputValueProps) {
+    const intl = useIntl();
+    const [search, setSearch] = React.useState<string | undefined>(undefined);
+
+    const { data } = useQuery<GQLManufacturersFilterQuery, GQLManufacturersFilterQueryVariables>(manufacturersQuery, {
+        variables: {
+            offset: 0,
+            limit: 10,
+            search: search,
```

Comment:
Should the `search` in the query be debounced?
Otherwise a request would be sent of every keystroke.